src/tests/__tests__/Checkbox.react-test.js

```diff
@@ -15,4 +14,12 @@ describe('Checkbox Component', () => {
     tree.props.checked = true;
     expect(tree).toMatchSnapshot();
   });
+
+  test('render', () => {
+    const component = renderer.create(
+      <Checkbox fancy />
+    );
+    let tree = component.toJSON();
```

Comment:
minor: since we're generally avoiding `let` and since the var is not used for anything else AND doesn't really add much to readability, would be best to just `expect(component.toJSON()).toMatchSnapshot()`